app/services/customization/customization.ts

```diff
@@ -31,7 +31,8 @@ export class CustomizationService
     previewSize: 300,
     livedockSize: 0.28,
     performanceMode: false,
-    chatZoomFactor: 1
+    chatZoomFactor: 1,
+    enableBBTVEmotes: false
```

Comment:
The code of including BBTTV emotes based on my reverse-engineering of BetterTTV plugin behavior. So I'm not sure that this code will work forever and prefer to disable this feature by default.